test/dynamo/test_dynamo.py

```diff
@@ -93,9 +93,11 @@ def test_resnet18(self):
       output_cpu = resnet18(data.cpu())
       self.assertTrue(
           torch.allclose(output_cpu, output.cpu(), rtol=1e-05, atol=1e-05))
-    # We only expect one graph for the resnet18 inference.
-    self.assertEqual(met.metric_data('CompileTime')[0], 1)
-    self.assertEqual(met.metric_data('ExecuteTime')[0], sample_count)
+    # One graph for fetching the fallback ops.
+    # Another graph for the resnet18 inference.
+    self.assertEqual(met.metric_data('CompileTime')[0], 2)
+    # Again, +1 offset in ExecuteTime for fetching the fallback ops.
+    self.assertEqual(met.metric_data('ExecuteTime')[0], sample_count + 1)
```

Comment:
ditto, we should not introduce additional exeuction for non-fallback graphs.

Reply:
Reverted this back with the latest commit that adds `ClearPendingIrs`.